backend/app/api/endpoints/kind/skill_marketplace.py

```
@@ -0,0 +1,363 @@
+        sort_order=sort_order,
+    )
+
+
+@router.get(
+    "/marketplace/{marketplace_skill_id}",
+    response_model=MarketplaceSkillDetailResponse,
+    summary="Get marketplace skill detail",
+)
+def get_marketplace_skill_detail(
+    marketplace_skill_id: int,
+    db: Session = Depends(get_db),
```

Ruff (0.14.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)